app/[locale]/ethereum-history-founder-and-ownership/page.tsx

```diff
@@ -0,0 +1,589 @@
+                      announcement: (chunks) => (
+                        <Link href="https://www.youtube.com/watch?v=l9dpjN3Mwps">
+                          {chunks}
+                        </Link>
+                      ),
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-4",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-5",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-6",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+                <ListItem>
+                  {t.rich(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-7",
+                    {
+                      strong: Strong,
+                    }
+                  )}
+                </ListItem>
+              </UnorderedList>
+              <CommentCard
+                description={t(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-8"
+                )}
+                name="Joseph Lubin"
+                title={t(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-9"
+                )}
+              />
+              <p>
+                {t(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-10"
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-11",
+                  {
+                    genesisBlock: (chunks) => (
+                      <Link href="https://etherscan.io/block/0">{chunks}</Link>
+                    ),
+                  }
+                )}
+              </p>
+              <div>
+                <LinkWithArrow href="/upgrades/">
+                  {t(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-12"
+                  )}
+                </LinkWithArrow>
+                <LinkWithArrow href="/roadmap/">
+                  {t(
+                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-13"
+                  )}
+                </LinkWithArrow>
```

Comment:
/upgrades/ forwards to /roadmap/... these link to the same place. Would switch the first one to the /history/ page, or I guess that's now `/ethereum-forks/`

```suggestion
                <LinkWithArrow href="/ethereum-forks/">
                  {t(
                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-12"
                  )}
                </LinkWithArrow>
                <LinkWithArrow href="/roadmap/">
                  {t(
                    "page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-13"
                  )}
                </LinkWithArrow>
```